core/src/epicli/data/common/ansible/playbooks/roles/helm/tasks/main.yml

```diff
@@ -0,0 +1,55 @@
+---
+- name: Get list of Helm repos 
+  become: yes
```

Comment:
Please consider setting `become: true` on higher level, this will remove repeating lines.